cloudflare.go

```diff
@@ -442,18 +448,21 @@ func (worker *CloudflareWorker) UpdateIPLists() error {
 			action = worker.Account.DefaultAction
 			worker.Logger.Debugf("ip action defaulted to %s", action)
 		}
-		for ip := range worker.CFStateByAction[action].IPListState.IPSet {
+		for ip, item := range worker.CFStateByAction[action].IPListState.IPSet {
 			if _, ok := newIPSetByAction[action]; !ok {
-				newIPSetByAction[action] = make(map[string]struct{})
+				newIPSetByAction[action] = make(map[string]IPSetItem)
 			}
-			newIPSetByAction[action][ip] = struct{}{}
+			newIPSetByAction[action][ip] = item
 		}
+
 		for _, decision := range decisions {
 			if _, ok := newIPSetByAction[action]; !ok {
-				newIPSetByAction[action] = make(map[string]struct{})
+				newIPSetByAction[action] = make(map[string]IPSetItem)
 			}
 			if _, ok := newIPSetByAction[action][*decision.Value]; !ok {
-				newIPSetByAction[action][*decision.Value] = struct{}{}
+				newIPSetByAction[action][*decision.Value] = IPSetItem{
```

Comment:
don't we want to keep the created_at of the decision itself ?

Reply:
LAPI doesn't provide created_at of the decision:
```
    {
      "duration": "145h18m14.813985321s",
      "id": 20231,
      "origin": "CAPI",
      "scenario": "crowdsecurity/ssh-bf",
      "scope": "Ip",
      "type": "ban",
      "value": "99.43.32.83"
    }
```